src/getOptions.js

```diff
@@ -34,6 +34,10 @@ function getOptions(loaderContext) {
     }
   }
 
```

Comment:
```js
if (typeof options.modifyVars === 'string') {
  try {
    loaderContext.addDependency(options.modifyVars);
    // Load variables from external file 
    options.modifyVars = require(options.modifyVars);
  } catch (err) {
    loaderContext.emitError(`Less Loader (modifyVars)\n\n${err.message}`);
  }  
}
```

Reply:
`require` will be cached and the reload will not work

Reply:
`delete require.cache[options.modifyVars]`

Reply:
```js
if (typeof options.modifyVars === 'string') {
  try {
    loaderContext.addDependency(options.modifyVars);
    // Clean the variables cache (require.cache)
    delete require.cache(options.modifyVars);
    // Load variables from external file 
    options.modifyVars = require(options.modifyVars);
  } catch (err) {
    loaderContext.emitError(new Error(`Less Loader (modifyVars)\n\n${err.message}`));
  }  
}
```

Reply:
If you don't want to include this addition in your PR it's also fine to leave as is :)

Reply:
good suggestion😀